packages/react/src/stories/ic-top-navigation.stories.mdx

```diff
@@ -476,7 +476,10 @@ export const DailyTippers = () => (
       ),
     ]}
   >
```

Comment:
Not directly relevant to this ticket. But I presume we need a ticket to implement this same behaviour to top navigation?

Reply:
Just to check on this, are you opening this ticket or was you wanting me too?

Reply:
Happy to create the ticket, do you mean that we need to implement it in the guidance site top navigation?

Reply:
Yes please!